packages/components/src/custom/Helpers/Dimension/windowSize.tsx

```diff
@@ -0,0 +1,46 @@
+import { useEffect, useState } from 'react';
+
+/**
+ *
+ * @returns {object} { width, height }
+ */
+function getWindowDimensions(): object {
```

Comment:
@sudhanshutech  can you define a `interface` or custom `type`  , rather than using only `object`